guides/common/modules/proc_viewing-vulnerabilities.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="viewing-vulnerabilities"]
+= Viewing vulnerabilities
+
+You can view Common Vulnerabilities and Exposures (CVE) from the {vulnerabilityengine}.
+You can examine how the CVEs affect your hosts.
+
+:FeatureName: {vulnerabilityengine}
+include::snip_technology-preview.adoc[]
+
+.Prerequisites
+* You have installed the {vulnerabilityengine}.
+* Your {Project} account has a role that grants one of the following permissions: `view_foreman_rh_cloud` or `view_insights_hits`.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Insights* > *Vulnerability*.
+. In the *CVE ID* column, click the ID of the vulnerability to see details.
+
+[role="_additional-resources"]
+.Additional resources
+* Installing {advisorengine} 
+// value: Insights advisor in Satellite
```

Comment:
@jeremylenz @ShimShtein The problem is that users don't install Advisor engine and Vulnerability engine separately but as one IoP component. Should we have a third variable that will cover the whole IoP experience?

Reply:
cc @evgeni ^^ We currently don't have a variable for the IoP. `{Insights}` refers to the cloud service AFAIK.